jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -259,7 +259,7 @@ def mos_spec1d_parser(app, data_obj, data_labels=None,
 
 @data_parser_registry("mosviz-spec2d-parser")
 def mos_spec2d_parser(app, data_obj, data_labels=None, add_to_table=True,
-                      show_in_viewer=False, ext=1, transpose=False):
+                      show_in_viewer=False, ext=1, transpose=False, cache=True):
```

Comment:
Why does the default to True, unlike others? Maybe should clarify in Mosviz import data section you added?